app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
```

Comment:
use same styling in your code, and compare it to proper stling by checklist:
https://github.com/NEkropo1/cafe-test-task-pt/blob/master/checklist.md